src/test/java/lotto/LottoNumberTest.java

```diff
@@ -2,16 +2,28 @@
 
 import lotto.domain.LottoNumber;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class LottoNumberTest {
 
     @Test
     void 보너스번호일치() {
-        LottoNumber lottoNumber = new LottoNumber(6);
-        LottoNumber bonusNumber = new LottoNumber(6);
+        LottoNumber lottoNumber = LottoNumber.of(6);
+        LottoNumber bonusNumber = LottoNumber.of(6);
 
         assertThat(lottoNumber.isSameNumber(bonusNumber)).isTrue();
     }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 46})
+    void 로또번호유효성체크(int number) {
+        assertThatThrownBy(() -> {
+            LottoNumber lottoNumber = LottoNumber.of(number);
```

Comment:
```suggestion
            LottoNumber.of(number);
```

따로 리턴값 없이 호출해도 좋을 것 같네요 😄